### DISCUSSION TITLE
Add information that someone you follow on GitHub is interested in the project that you're reviewing

### DISCUSSION THREAD

**Discussion by olekstomek** [community/community#20637] [Other Feature Feedback, Questions, & Ideas]
Idea - you browse a repository and you can see information about the repository, such as: 
- who contributes, 
- who give the star, 
- who make the project fork,
- who watches the project, 
- who supports the project by money.

How about the idea that while on the repository's website you get information that **someone you follow on GitHub**: contributes to the project and/or gives a star and/or forks the project and/or observes the project and/or supports the project.

If the person who is following by you gets involved or has in some way shown interest in the project in the repository, this may be a piece of information for you that it is worth taking a look at this project.

See that, for example, on Instagram when entering someone's profile, you have information about who of your friends is watching this profile.

**Comment by jfullstackdev** [comment]
it is somewhat implemented on your feed, when a user who you are following starred a repo or contributed but not automatically on the repo, but you will be seeing that on the Contributors or say the watchers list

so after seeing that on your feed, then you may check the repo further

**Reply by olekstomek** [reply]
In case someone did some action (let's say a long time ago) on the repository before I started following him then I don't have that information on my feed.
In addition, this information is only available at the time of performing an action on the repository.
